webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala

```diff
@@ -1816,6 +1816,12 @@ class OntologyResponderV2(responderData: ResponderData) extends Responder(respon
                 // Check that the ontology exists and has not been updated by another user since the client last read its metadata.
                 _ <- checkOntologyLastModificationDateBeforeUpdate(internalOntologyIri = internalOntologyIri, expectedLastModificationDate = changeOntologyMetadataRequest.lastModificationDate)
 
+                // get the metadata of the ontology.
+                existingOntologyMetadata: Option[OntologyMetadataV2] <- loadOntologyMetadata(internalOntologyIri)
```

Comment:
You don't want to load the ontology metadata from the triplestore here. It's already in the cache. For performance reasons, ontologies are loaded when Knora starts.

Reply:
See the ontology responder design doc here: https://docs.knora.org/05-internals/design/api-v2/ontology-management/

Reply:
Thank you for pointing this out. I just fixed this.